staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -270,3 +270,13 @@ func (list ErrorList) Filter(fns ...utilerrors.Matcher) ErrorList {
 	// FilterOut takes an Aggregate and returns an Aggregate
 	return fromAggregate(err.(utilerrors.Aggregate))
 }
+
+// Is reports whether any Error in err's chain matches target.
+func (e *Error) Is(target error) bool {
+
+	t, ok := target.(*Error)
+	if !ok {
+		return false
+	}
+	return (e.Type == t.Type && e.Field == t.Field && e.BadValue == t.BadValue && e.Detail == t.Detail)
```

Comment:
Parentheses aren't neeeded. Also, maybe it would be more readable to have each condition on a separate line.